docs/README_tl.md

```diff
@@ -0,0 +1,323 @@
+  <a href="https://www.prisma.io"><img alt="Prisma" src="https://img.shields.io/badge/-Prisma-1B222D?style=flat-square&logo=prisma&logoColor=white" /></a>
+  <a href="https://www.mongodb.com"><img alt="MongoDB" src="https://img.shields.io/badge/-MongoDB-47A248?style=flat-square&logo=mongodb&logoColor=white" /></a>
+  <a href="https://www.postgresql.org"><img alt="PostgreSQL" src="https://img.shields.io/badge/-PostgreSQL-657991?style=flat-square&logo=postgresql&logoColor=white" /></a>
+  <a href="https://www.mysql.com"><img alt="MySQL" src="https://img.shields.io/badge/-MySQL-00688F?style=flat-square&logo=mysql&logoColor=white" /></a>
+</p>
+
+API na nagbibigay ng impormasyon tungkol sa **Mga administratibong lugar sa Indonesia**, sa antas ng lalawigan, bayan, distrito, hanggang sa mga barangay. Isinasama rin nito ang datos tungkol sa mga isla simula [version 1.1.0](https://github.com/fityannugroho/idn-area/releases/tag/v1.1.0).
+
+Ginawa gamit ang [NestJS framework](https://nestjs.com) At isinulat sa TypeScript.[Prisma](https://www.prisma.io) ay ginagamit bilang ORM upang makipag-ugnay sa anumang uri ng mga database(MySQL, PostgreSQL, and MongoDB).
+
+> **⚠️ Upgrading to v3.0.0 ⚠️**
+>
+> Since [v3.0.0](https://github.com/fityannugroho/idn-area/releases/tag/v3.0.0), **Node.js v18** or higher is required.
+
+<h2>Talaan ng Nilalaman</h2>
+
+- [Pagsisimula](#getting-started)
+- [Data](#data)
+- [API Dulopunto](#api-endpoint)
+  - [Dokumentasyon](#documentation)
+  - [Kumuha probinsya](#get-provinces)
+  - [Kumuha partikular na probinsya](#get-specific-province)
+  - [Kuhaan ng mga lungsod ayon sa pangalan](#get-regencies-by-name)
+  - [Kumuha partikular na bayan](#get-specific-regency)
+  - [Kumuha lahat ng mga bayan sa isang lalawigan](#get-all-regencies-in-a-province)
+  - [Kumaha ng mga distrito ayon sa pangalan](#get-districts-by-name)
+  - [Kumuha partikular distrito](#get-specific-district)
+  - [Kuhaan lahat ng mga distrito sa isang bayan](#get-all-districts-in-a-regency)
+  - [Kuhaan ng mga barangay ayon sa pangalan](#get-villages-by-name)
+  - [Kumuha partikular barangay](#get-specific-village)
+  - [Kuhaan lahat ng mga barangay sa isang distrito](#get-all-villages-in-a-district)
+  - [Kuhaan ng mga isla ayon sa pangalan](#get-islands-by-name)
+  - [Kuhaan ng tiyak na isla](#get-specific-island)
+  - [Kuhaan lahat ng mga isla sa isang bayan](#get-all-islands-in-a-regency)
+- [Query Parameters](#query-parameters)
+  - [`sortBy`](#sortby)
+  - [`sortOrder`](#sortorder)
+- [Live na pagpapakita](#live-demo)
+- [Nagbibigay ng kontribusyon](#contributing)
+- [Ulat ng Problema](#problem-reporting)
+  - [Pag-uulat ng Isang Bug](#reporting-a-bug)
+  - [Humihiling ng Bagong Tampok](#requesting-a-new-feature)
+  - [Nagtatanong ng Tanong](#asking-a-question)
+- [Suportahan ang Proyektong Ito](#support-this-project)
```

Comment:
The links in the table of content (TOC) also need to be translated to make it work. The link is a **snake-case** version of the header **without any symbol**, for example:

| Header | Link |
|--------|--------|
| Suportahan ang Proyektong Ito | #suportahan-ang-proyektong-ito |
| Nagtatanong ng Tanong | #nagtatanong-ng-tanong | 

So that when the reader clicks a link on the TOC, the page will scroll down to the intended section.